src/language-css/printer-postcss.js

```diff
@@ -679,6 +679,7 @@ function genericPrint(path, options, print) {
               )
             ])
           ),
+          node.groups.length > 1 && options.trailingComma === "es5" ? "," : "",
```

Comment:
Sorry, I was unclear in my first review. We should print a comma here if `options.trailingComma !== "none"`, since we don’t care which option the user picks in this case (`all` or `es5`).